src/ai/backend/agent/kubernetes/kernel.py

```diff
@@ -511,7 +511,7 @@ async def copy_runner_files(scratch_path: Path) -> None:
     shutil.copytree(helpers_path.resolve(), destination_path / "helpers")
 
 
-async def prepare_krunner_env(local_config: Mapping[str, Any]) -> Mapping[str, Sequence[str]]:
+async def prepare_krunner_env(local_config: Mapping[str, Any]) -> Mapping[str, str]:
```

Comment:
Was the type that was actually being used being returned as str?

Reply:
yes, the type annotation has been wrong here